Assignment2/index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+require './creds.php';
```

Comment:
Let's do this in a different file and try to segregate html and php code.
Addtionally please implement OOps